llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -5604,6 +5676,14 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
       break;
     }
 
+    // AVX Galois Field New Instructions
```

Comment:
Why new?

Reply:
Because the 80386 did not have Galois Field instructions.
(Intel officially refers to "Galois Field New Instructions"; and support for these instructions is via the "GFNI" feature flag in CPUID.)